robot_control_components/src/effort_actuator.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 ROS2-Control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "hardware_interface/component_interfaces/actuator_interface.hpp"
+
+namespace robot_control_components
+{
+
+using hardware_interface::hardware_interface_ret_t;
+using hardware_interface::HW_RET_OK;
+using hardware_interface::ComponentInfo;
+
+class EffortActuator : public hardware_interface::ActuatorInterface
+{
+public:
+  EffortActuator() = default;
+
+  ~EffortActuator() = default;
+
+  hardware_interface_ret_t configure(const ComponentInfo & actuator_info)
```

Comment:
if you don't specify the parameter name in the cpp, the linter won't cry about the unused parameter and we save on some boilerplate

```suggestion
  hardware_interface_ret_t configure(const ComponentInfo & /* actuator_info */)
  {
    return hardware_interface::HW_RET_OK;
  }
```

Reply:
I would roll with the `(void) arg` for now. We can re-iterate over these in a dedicated PR all over the existing code base.

Reply:
Agreed and I love the new "create an issue from this" feature of github. https://github.com/ros-controls/ros2_control/issues/115